pkg/providers/builtin/bigtable/definition.go

```diff
@@ -100,11 +101,65 @@ func ServiceDefinition() *broker.ServiceDefinition {
 				FieldName: "zone",
 				Type:      broker.JsonTypeString,
 				Details:   "The zone to create the Cloud Bigtable cluster in. Zones that support Bigtable instances are noted on the Cloud Bigtable locations page: https://cloud.google.com/bigtable/docs/locations.",
-				Default:   "us-east1-b",
```

Comment:
as discussed:
- revert default
- check constraints if should be also reverted

Reply:
Each `enum` field type does not have `constraints` parameter